### PR TITLE
Add python line to top of pizzly scripts.

### PR DESCRIPTION
**Pull request by roryk**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

This makes the scripts executable and adds them to the tests.

### CODE REVIEW

File: recipes/pizzly/build.sh [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 cp pizzly $PREFIX/bin
 wget https://raw.githubusercontent.com/pmelsted/pizzly/master/scripts/flatten_json.py -O $PREFIX/bin/pizzly_flatten_json.py
+sed -i.bak "1s@^@#!/usr/bin/env python\'$'\n@g" $PREFIX/bin/pizzly_flatten_json.py
 chmod +x $PREFIX/bin/pizzly_flatten_json.py
-wget https://github.com/pmelsted/pizzly/blob/master/scripts/get_fragment_length.py -O $PREFIX/bin/pizzly_get_fragment_length.py
-chmod +x $PREFIX/bin/pizzly_flatten_json.py
+wget https://raw.githubusercontent.com/pmelsted/pizzly/master/scripts/get_fragment_length.py -O $PREFIX/bin/pizzly_get_fragment_length.py
```
Comment (bgruening) [inline recipes/pizzly/build.sh]: I'm surprised that this is still needed.

File: recipes/pizzly/build.sh [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
+sed -i.bak "1s@^@#!/usr/bin/env python\'$'\n@g" $PREFIX/bin/pizzly_get_fragment_length.py
+chmod +x $PREFIX/bin/pizzly_get_fragment_length.py
```
Comment (bgruening) [inline recipes/pizzly/build.sh]: I'm surprised that this is still needed.

File: recipes/pizzly/build.sh [hunk 3]
```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 cp pizzly $PREFIX/bin
 wget https://raw.githubusercontent.com/pmelsted/pizzly/master/scripts/flatten_json.py -O $PREFIX/bin/pizzly_flatten_json.py
+sed -i.bak "1s@^@#!/usr/bin/env python\'$'\n@g" $PREFIX/bin/pizzly_flatten_json.py
 chmod +x $PREFIX/bin/pizzly_flatten_json.py
-wget https://github.com/pmelsted/pizzly/blob/master/scripts/get_fragment_length.py -O $PREFIX/bin/pizzly_get_fragment_length.py
-chmod +x $PREFIX/bin/pizzly_flatten_json.py
+wget https://raw.githubusercontent.com/pmelsted/pizzly/master/scripts/get_fragment_length.py -O $PREFIX/bin/pizzly_get_fragment_length.py
+sed -i.bak "1s@^@#!/usr/bin/env python\'$'\n@g" $PREFIX/bin/pizzly_get_fragment_length.py
```
Comment (bgruening) [inline recipes/pizzly/build.sh]: This is replacing the `import sys` I think.

File: recipes/pizzly/build.sh [hunk 4]
```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 cp pizzly $PREFIX/bin
 wget https://raw.githubusercontent.com/pmelsted/pizzly/master/scripts/flatten_json.py -O $PREFIX/bin/pizzly_flatten_json.py
+sed -i.bak "1s@^@#!/usr/bin/env python\'$'\n@g" $PREFIX/bin/pizzly_flatten_json.py
```
Comment (bgruening) [inline recipes/pizzly/build.sh]: This is probably replacing the `import sys` at the first line.

File: recipes/pizzly/build.sh (hunk 1 above)
Comment (bgruening) [inline recipes/pizzly/build.sh]: Also we should pin this to a specific commit.

**Comment by roryk** [comment]
Thanks @bgruening-- I dunno how the tests passed locally. I fixed the addition of the shebang and pinned the downloads to a blob.

File: recipes/pizzly/build.sh [hunk 5]
```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 cp pizzly $PREFIX/bin
-wget https://raw.githubusercontent.com/pmelsted/pizzly/master/scripts/flatten_json.py -O $PREFIX/bin/pizzly_flatten_json.py
-chmod +x $PREFIX/bin/pizzly_flatten_json.py
-wget https://github.com/pmelsted/pizzly/blob/master/scripts/get_fragment_length.py -O $PREFIX/bin/pizzly_get_fragment_length.py
+blob=251989979ba2cb39e217a91aa7fefc85c554ebe1
+wget https://raw.githubusercontent.com/pmelsted/pizzly/${blob}/scripts/flatten_json.py -O $PREFIX/bin/pizzly_flatten_json.py
+sed -i.bak '1 i\#!/usr/bin/env python -Es' $PREFIX/bin/pizzly_flatten_json.py
```
Comment (bgruening) [inline recipes/pizzly/build.sh]: This seems not to be OSX compliant :(

**Comment by roryk** [comment]
You know what they say, 6th try is a charm.

**Pull request merged**